docs/admin/install.rst

```diff
@@ -649,7 +649,7 @@ The :file:`settings.py` snippet for MySQL:
             'OPTIONS': {
                 # In case of older MySQL server which has default MariaDB
```

Comment:
```suggestion
                # In case of using an older MySQL server, which has MariaDB by default
```
Does it though?

Reply:
The original comment is wrong, it should be talking about MyISAM instead of MariaDB. Fixed in 418673d62a